apollo-router/src/configuration/mod.rs

```diff
@@ -23,7 +23,7 @@ use regex::Regex;
 use rustls::pki_types::CertificateDer;
 use rustls::pki_types::PrivateKeyDer;
 use rustls::ServerConfig;
-use schemars::gen::SchemaGenerator;
+use schemars::r#gen::SchemaGenerator;
```

Comment:
If we can't change this now then we should create a TODO (or GitHub issue/Jira task) to rename it to `generator` in a later version.